.github/workflows/test-integration-main.yaml

```diff
@@ -117,6 +118,7 @@ jobs:
         kubectl label ns $TEST_NAMESPACE github-job-id=$GITHUB_WORKFLOW_JOB_ID
         kubectl label ns $TEST_NAMESPACE github-pr-id=${{ github.event.pull_request.number }}
         kubectl label ns $TEST_NAMESPACE test-persistent=${{ env.CAMUNDA_DISTRO_TEST_PERSISTENT }}
+        echo "TEST_NAMESPACE=$TEST_NAMESPACE" >> "$GITHUB_OUTPUT"
```

Comment:
no need to do that since the namespace is already exposed inside the test pod (env name `K8S_NAMESPACE`).